stable_baselines/agent.py

```diff
@@ -4,13 +4,10 @@
 import yaml
 import json
 import argparse
-base_path = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(os.path.join(base_path, '../'))
-from make_sb_env import make_sb_env
-
+from diambra.arena.stable_baselines.make_sb_env import make_sb_env
 from stable_baselines import PPO2
 
-# diambra run -g python stable_baselines/v2/scripts/evaluation.py --cfgFile $PWD/stable_baselines/v2/scripts/cfg_files/doapp/sr6_128x4_das_nc.yaml  --trainedModel "25M"
+# diambra run -g python stable_baselines/agent.py --cfgFile $PWD/stable_baselines/cfg_files/doapp/sr6_128x4_das_nc.yaml  --trainedModel "25M"
```

Comment:
What about moving this with a few more words to a docstring at the top of the file?
```
"""This is a example agent based on stable baselines.

Usage:
diambra run -g python stable_baselines/agent.py --cfgFile $PWD/stable_baselines/cfg_files/doapp/sr6_128x4_das_nc.yaml --trainedModel "25M"
"""

Reply:
(I think it's just better for consistency but we can also use that eventually to generate docs from it automatically)